custom_components/tahoma/const.py

```diff
@@ -56,8 +56,9 @@
 }
 
 TAHOMA_HORIZONTAL_DEVICES = {
-    "PositionableHorizontalAwning",
-    "UpDownHorizontalAwning"
+    "PositionableHorizontalAwning": DEVICE_CLASS_AWNING,
+    "PositionableHorizontalAwningUno": DEVICE_CLASS_AWNING,
+    "UpDownHorizontalAwning": DEVICE_CLASS_AWNING
```

Comment:
The DEVICE_CLASS_AWNING is not required, it is already set in the part above by the `Awning` device type. This dict is just for enabling specific Horizontal features, because those can not be enabled based on the widget or uiclass unfortunately.

Reply:
I thought the device class is missing because it didn't work at the beginning. But probably there war only the widget type `PositionableHorizontalAwningUno` missing.